tools/accuracy_checker/accuracy_checker/annotation_converters/image_processing.py

```diff
@@ -58,22 +62,30 @@ def configure(self):
         self.data_dir = self.get_value_from_config('data_dir')
         self.in_suffix = self.get_value_from_config('input_suffix')
         self.out_suffix = self.get_value_from_config('target_suffix')
+        self.recursive = self.get_value_from_config('recursive')
         self.annotation_loader = LOADERS_MAPPING.get(self.get_value_from_config('annotation_loader'))
         if not self.annotation_loader:
             raise ConfigError('provided not existing loader')
 
     def convert(self, check_content=False, progress_callback=None, progress_interval=100, **kwargs):
         content_errors = [] if check_content else None
         file_list_in = []
-        for file_in_dir in self.data_dir.iterdir():
+        if self.recursive:
+            data_dir_files = [file for file in self.data_dir.glob('**/*') if file.is_file()]
```

Comment:
```suggestion
            data_dir_files = [file for file in self.data_dir.rglob('*') if file.is_file()]
```
https://docs.python.org/3/library/pathlib.html#pathlib.Path.rglob